src/uwtools/resources/FV3Forecast.jsonschema

```diff
@@ -90,41 +221,54 @@
           "type": "string"
         }
       },
+      "required": [
+        "scheduler"
+      ],
```

Comment:
Is a scheduler required at the YAML level? I don't think we would need that if we're running without the `--batch-script` flag. We do always need the `mpicmd`, though.

Reply:
It doesn't look like `scheduler` is ever accessed unless `--batch_script` (CLI) or `batch_script` (API) is set. Will fix.

`mpicmd` is optional in the code:
```
self._platform_config.get("mpicmd")
```

Should that change to
```
self._platform_config["mpicmd"]
```

?

Reply:
I proactively made `mpicmd` required in both schema and code, but please let me know if that's not right.

Reply:
Having `mpicmd` required both in code and in the schema is probably the right move. I don't think FV3 could run without some MPI command.

Reply:
`mpicmd` is now required, in schema and in code.